tests/trx_generator/main.cpp

```diff
@@ -173,7 +157,7 @@ vector<name> get_accounts(const vector<string>& account_str_vector) {
 
 vector<fc::crypto::private_key> get_private_keys(const vector<string>& priv_key_str_vector) {
    vector<fc::crypto::private_key> key_list;
-   for(string private_key : priv_key_str_vector) {
+   for(string private_key: priv_key_str_vector) {
```

Comment:
Use reference to avoid the copy.

Reply:
Addressed: 42c5900bc2bc06b14da3f3879778b476b6f86c3f